### DISCUSSION TITLE
Что означает ошибка "split cancel:"?

### DISCUSSION THREAD

**Discussion by Lurker00** [hufrea/byedpi#129] [Q&A] [answered]
Диагностика делается тут:

https://github.com/hufrea/byedpi/blob/dc6966ec1c4d6cdb436ca5ce22d99fb51ec5259d/desync.c#L503-L504

В логах таких строк бывает много. В каких ситуациях это происходит? Стоит ли что-то предпринимать против этого, и если да - что? А если не надо - может, перевести в разряд `LOG_S`?

**Comment by hufrea** [comment]
> В каких ситуациях это происходит? 

Позиция меньше предыдущей или больше размера полученных данных.

> Стоит ли что-то предпринимать против этого, и если да - что? 

https://github.com/hufrea/byedpi/blob/dc6966ec1c4d6cdb436ca5ce22d99fb51ec5259d/README.md?plain=1#L149
Указать параметры в другом порядке. `-s 2 -d 1` -> `-d 1 -s 2`

**Comment by Lurker00** [comment]
А у меня нет параметра `-s` (`--split`):
```
ARGS="--transparent --port $PORT --proto=http --auto=none --hosts $HOSTS --disorder 1 --fake 0 --ttl 1 --auto=torst --hosts $HOSTS --tlsrec 1+s"
```
Параметры копипастил откуда-то отсюда. Поменять местами `--fake` и `--disorder`?

**Reply by hufrea** [reply, marked as answer]
Удалить `--fake` и `--ttl`,  они и так не работали. К тому же, с правильным порядком это и так было бы бессмысленным — `--fake 0` отправит 0 байт, т.е. ничего. Да даже если бы было указано другое значение, из-за `--ttl 1` пакет все равно бы до DPI не дошел.

**Reply by Lurker00** [reply]
Похоже, получилось!
